csharp-basic-topics/TypesOfInheritanceInCSharp/Tests/TypesOfInheritanceTest.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.IO;
+using TypesOfInheritanceInCSharp;
+using Xunit;
+
+namespace TypesOfInheritanceInCSharpTests;
+public class TypesOfInheritanceTest
```

Comment:
Newline

Reply:
It seems to me that this blank line is already added (line 5)